tests/addressing/user.py

```diff
@@ -23,7 +23,7 @@
 LOGGER = logging.getLogger(__name__)
 
 
-@pytest.mark.unit
+@pytest.mark.library
```

Comment:
Not a show-stopper, but I am not seeing how we will be adopting/applying this at this time. 

What I can see is using pytest decorators to filter out tests we would run in a build pipeline (setting aside slow running tests for downstream/e2e tests to improve pre-merge build time). We can employ the decoration once it solves an existing problem. Without a problem being solved comes a reduced value-added.